frontend/src/metabase/entities/containers/EntityObjectLoader.jsx

```diff
@@ -74,7 +74,11 @@ class EntityObjectLoaderInner extends React.Component {
     ) {
       nextProps.fetch(
         { id: nextProps.entityId, ...nextProps.entityQuery },
-        { reload: nextProps.reload, properties: nextProps.properties },
+        {
+          reload: nextProps.reload,
+          properties: nextProps.properties,
+          noEvent: !nextProps.dispatchApiErrorEvent,
```

Comment:
This fixes a bug where the `dispatchApiErrorEvent` prop wasn't being followed through, which was inconsistent with line 65. It caused the 403 error page to show incorrectly when the user didn't have access to a referenced card in a template tag of a native query they were opening.
https://github.com/metabase/metabase/blob/173ecb8f23f90501d0bad106102ae7cf205e984d/frontend/src/metabase/entities/containers/EntityObjectLoader.jsx#L65